src/components/member/tabs/contract-tab/FromDateComponent.tsx

```diff
@@ -0,0 +1,71 @@
+import { Agreement, Contract } from 'api/generated/graphql'
+import { format } from 'date-fns'
+import { Button, ButtonsGroup } from 'hedvig-ui/button'
+import { DateTimePicker } from 'hedvig-ui/date-time-picker'
+import * as React from 'react'
+import { Card } from '../../../../../shared/hedvig-ui/card'
+import {
+  changeFromDateOptions,
+  useChangeFromDate,
+} from '../../../../graphql/use-change-from-date-for-agreement'
+import { ButtonSpacing } from './contract-item'
+
+const initialFromDate = (agreement: Agreement): Date =>
+  agreement.fromDate ? new Date(agreement.fromDate) : new Date()
+
+const FromDateComponent: React.FunctionComponent<{
+  agreement: Agreement
+  contract: Contract
+}> = ({ agreement, contract }) => {
+  const [datePickerEnabled, setDatePickerEnabled] = React.useState(false)
+  const [fromDate, setFromDate] = React.useState(initialFromDate(agreement))
+  const [
+    changeFromDate,
+    { loading: changeFromDateLoading },
+  ] = useChangeFromDate(contract)
+  const reset = () => {
+    setFromDate(initialFromDate(agreement))
+    setDatePickerEnabled(false)
+  }
+
+  return (
+    <>
+      <Card span={2}>
```

Comment:
This Card should probably be in the parent component and this component should be the content inside the cardm that way to decide the spacing/span/layout from the parent component.
```suggestion
      <>
```